index.html

```diff
@@ -138,11 +138,11 @@ <h3 id="form-header">Get Started</h3>
           method="POST"
           >
           <label for="name"></label>
-          <input class="st-input" type="text" name="name" placeholder="Your name" maxlength="30" required>
+          <input class="st-input" type="text" name="name" placeholder="Your name" required maxlength="30">
           <label for="email"></label>
           <input class="st-input" type="email" name="email" placeholder="Email address" required>
           <label for="message"></label>
-          <textarea name="message" type="text" maxlength="500" placeholder="Add your message here..." required></textarea>
+          <textarea name="message" type="text" placeholder="Add your message here..." required maxlength="500"></textarea>
           <!-- your other form fields go here -->
           <button class="btn-submit" type="submit">Start collaboration</button>

```

Comment:
Good job adding hover to your submit button. But let's make sure the color matches the provided design. The main goal of this project is to code the given design pixel perfectly. So let's try to match the design as much as possible and fix the hover color and add pressed state to our button. 
<img width="342" alt="Screenshot 2023-01-24 at 8 04 13 PM" src="https://user-images.githubusercontent.com/25887253/214365681-0328df53-e43c-4ea6-b1ea-a7760736d709.png">
<img width="406" alt="Screenshot 2023-01-24 at 8 14 42 PM" src="https://user-images.githubusercontent.com/25887253/214365685-9e8543cd-ecb2-4420-b02c-c8c23a9b7090.png">

Reply:
Done.